NTupleMaker/interface/functionsSynch2017.h

```diff
@@ -406,8 +406,22 @@ bool extra_electron_veto(int leptonIndex, TString ch, const Config *cfg, const A
     if (fabs(analysisTree->electron_dxy[ie])>=cfg->get<float>("dxyVetoElectronCut")) continue;
     if (fabs(analysisTree->electron_dz[ie])>=cfg->get<float>("dzVetoElectronCut")) continue;
 
-    bool electronMvaId = analysisTree->electron_mva_wp90_general_Spring16_v1[ie]; //analysisTree->electron_mva_wp90_nontrig_Synch17_v1[ie];
-    if (!electronMvaId && cfg->get<bool>("applyVetoElectronId")) continue;
+    int era=cfg->get<int>("era");
+    bool electronMvaId;
+
+     if (era==2016) {
+         electronMvaId = analysisTree->electron_mva_wp90_general_Spring16_v1[ie]>0.5; 
+      }
```

Comment:
For the legacy 2016 dataset the electron MVA used is electron_mva_wp90_noIso_Fall17_v1. You correctly identified that the one previously stored was outdated, but the old one will not be used unless there is a specific reason to do so.
Otherwise we will use the newest one also for legacy 2016.

Reply:
Hi Andrea, I suggest to resolve this in the revision campaign for 2016 synch ntupler, and implement when we know what data set we will use for 2016.